assets/bower.json

```diff
@@ -43,7 +44,8 @@
   },
   "appPath": "app",
   "resolutions": {
-    "lodash": "3.10.1"
+    "lodash": "3.10.1",
+    "fontawesome": "4.5.0"
```

Comment:
@jwforres, good to know.  I've submitted [a feature request to PatternFly to upgrade Font Awesome to v4.5.0](https://github.com/patternfly/patternfly/issues/213) so we don't need both versions of Font Awesome.